File: dart/pangram/lib/pangram.dart
class Pangram {
  var alphabet = "abcdefghijklmnopqrstuvwxyz".split('');
  bool isPangram(String input) {
    Set<String> formattedInput = input.toLowerCase().split('').toSet();
    bool pangram = true;
    int index = 0;
    while (index < alphabet.length) {
      String letter = alphabet[index];
      if (!(formattedInput.contains(letter))) {
        pangram = false;
        break;
      }
      index++;
    }
    return pangram;
  }
}

File: csharp/pangram/Pangram.cs
using System;

public static class Pangram
{
    public static bool IsPangram(string input)
    {
        string allCharString = "abcdefghijklmnopqrstuvwxyz";

        foreach (var character in allCharString)
        {
            if (!(input.ToLower().Contains(character)))
            {
                return false;
            }
        }

        return true;
    }
}
